### PR TITLE
Change to work for Brussels too

### PR DESCRIPTION
**Pull request by wpoely86**
I should probably add some tests too.

I've renamed the script from `sync_ugent_vsc_users.py` to `sync_vsc_users.py`. If it's too much hassle, I can keep the original name.

### CODE REVIEW

**Comment by wdpypere** [comment]
@wpoely86 if you change the name you should probably open a PR in the quattor tree to update the name there as well.

**Comment by wdpypere** [comment]
the same for the NAGIOS header change

File: test/ldapsync.py [hunk 1]
```diff
@@ -116,3 +117,7 @@ def test_sync_altered_groups(self, mock_add_or_update):
         ldap_attrs =  {'status': ['active'], 'cn': 'vsc40075', 'institute': ['gent'], 'memberUid': ['vsc40075'],
                        'moderator': ['vsc40075'], 'gidNumber': ['2540075']}
         mock_add_or_update.assert_called_with(VscLdapGroup, test_group.vsc_id, ldap_attrs, True)
+
+
+if __name__ == '__main__':
```
Comment (stdweird) [inline test/ldapsync.py]: what is this about?

Comment (wpoely86) [reply test/ldapsync.py]: being able to run the test directly.

Comment (stdweird) [reply test/ldapsync.py]: you don't need this to do so; `python setup.py test --help` there's options to select test module and function

Comment (wpoely86) [reply test/ldapsync.py]: That doesn't work as `vsc-ldap-extension` is not available (see #77). And it's faster.

Comment (stdweird) [reply test/ldapsync.py]: we should somehow fix the issue, but we are not going to pollute all unittests files with this.

Comment (stdweird) [reply test/ldapsync.py]: and also ` python -m unittest test.slurm` etc etc

**Comment by wpoely86** [comment]
@stdweird This is ready. Not yet tested with a live test.

File: lib/vsc/administration/user.py [hunk 2]
```diff
@@ -456,13 +466,13 @@ def process_users_quota(options, user_quota, storage_name, client, host_institut
         user.dry_run = options.dry_run
 
         try:
-            if storage_name in ['VSC_HOME']:
+            if storage_name in VSC_HOME:
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: `in` -> `==`

File: lib/vsc/administration/user.py [hunk 3]
```diff
@@ -456,13 +466,13 @@ def process_users_quota(options, user_quota, storage_name, client, host_institut
...
                 user.set_home_quota()
 
-            if storage_name in ['VSC_DATA']:
+            if storage_name in VSC_DATA:
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: `in` --> `==`

File: lib/vsc/administration/user.py [hunk 4]
```diff
@@ -499,15 +509,15 @@ def process_users(options, account_ids, storage_name, client, host_institute=Non
         user.dry_run = options.dry_run
 
         try:
-            if storage_name in ['VSC_HOME']:
+            if storage_name in VSC_HOME:
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: in -> ==

File: lib/vsc/administration/user.py [hunk 5]
```diff
@@ -499,15 +509,15 @@ def process_users(options, account_ids, storage_name, client, host_institute=Non
...
                 user.create_home_dir()
                 user.populate_home_dir()
                 update_user_status(user, client)
 
-            if storage_name in ['VSC_DATA']:
+            if storage_name in VSC_DATA:
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: in -> ==

File: setup.py [hunk 6]
```diff
@@ -24,15 +24,15 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '1.6',
+    'version': '1.6.1',
```
Comment (stdweird) [inline setup.py]: why change the numbering scheme?

File: lib/vsc/administration/user.py [hunk 7]
```diff
@@ -402,13 +413,12 @@ def __setattr__(self, name, value):
         super(VscTier2AccountpageUser, self).__setattr__(name, value)
 
 
-
 cluster_user_pickle_location_map = {
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: these are module constants? move them up, and use all caps
also, this need some vub support too?

File: lib/vsc/administration/user.py [hunk 8]
```diff
@@ -402,13 +413,12 @@ def __setattr__(self, name, value):
...
     'kyukon': VscTier2AccountpageUser,
 }
 
 cluster_user_pickle_store_map = {
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: same as above

File: lib/vsc/administration/user.py [hunk 9]
```diff
@@ -136,6 +138,18 @@ def __init__(self, user_id, storage=None, pickle_storage='VSC_SCRATCH_KYUKON', r
...
+
+        if host_institute is None:
+            host_institute = GENT
+
+        if pickle_storage is None:
+            pickle_storage = default_pickle_storage[host_institute]
+
         self.pickle_storage = pickle_storage
         if not storage:
             self.storage = VscStorage()
```
Comment (stdweird) [inline lib/vsc/administration/user.py]: i would add 2 extra attrs (and even get rid of `self.storage`)
```
self.institute_path_templates = self.storage.path_templates[self.host_institute]
self.institute_storage = self.storage[self.host_institute]
```
and use those below

File: setup.py (hunk 6 above)
Comment (wpoely86) [reply setup.py]: to make it consistent again. @itkovian said he forgot the `.0` in his last PR.

File: lib/vsc/administration/user.py (hunk 8 above)
Comment (wpoely86) [reply lib/vsc/administration/user.py]: This needs to be here as `VscTier2AccountpageUser` first needs to be defined.

File: lib/vsc/administration/user.py (hunk 7 above)
Comment (wpoely86) [reply lib/vsc/administration/user.py]: I'm not sure actually. They are not used anywhere in vsc-administration. @itkovian ?

Comment (wpoely86) [reply lib/vsc/administration/user.py]: @itkovian says this is legacy stuff which he wants to get rid of sooner rather then later. So I'm not sure if there is any point in looking this at VUB side?

Comment (stdweird) [reply lib/vsc/administration/user.py]: ok, well this is up to you.

**Comment by stdweird** [comment]
i'll let @itkovian merge this, so he is aware of the changes

**Pull request merged**